app/src/main/java/app/revanced/integrations/twitch/settingsmenu/TwitchPreferenceFragment.java

```diff
@@ -1,6 +1,18 @@
 package app.revanced.integrations.twitch.settingsmenu;
 
+import app.revanced.integrations.shared.Logger;
 import app.revanced.integrations.shared.settingsmenu.AbstractPreferenceFragment;
+import app.revanced.integrations.twitch.settings.Settings;
 
 public class TwitchPreferenceFragment extends AbstractPreferenceFragment {
+
+    @Override
+    protected void initialize() {
+        super.initialize();
+
+        // Do anything that forces this apps Settings bundle to load.
+        if (Settings.BLOCK_VIDEO_ADS.get()) {
```

Comment:
I understood whats happening here: using the reference here causes the `Settings` class of Twitch to be loaded. Is that correct?